unit/src/HitTests.C

```diff
@@ -314,6 +316,38 @@ TEST(ExpandWalkerTests, All)
        "hello/boo",
        "baz",
        hit::Field::Kind::String},
+      {"multi-line brace expression",
+       "foo=${raw 4\n"
+       "          2\n"
+       "     }",
+       "foo",
+       "42",
+       hit::Field::Kind::String},
+      {"fparse", "foo=${fparse 40 + 2}\n", "foo", "42", hit::Field::Kind::Float},
+      {"fparse-with-var", "var=39 foo=${fparse var + 3}", "foo", "42", hit::Field::Kind::Float},
+      {"brace-expression-ends-before-newline",
+       "foo=${raw 42} bar=23",
+       "bar",
+       "23",
+       hit::Field::Kind::Int},
+      {"super-complicated",
+       "foo1 = 42\n"
+       "foo2 = 43\n"
+       "[section1]\n"
+       "  num = 1\n"
+       "  bar = ${${raw foo ${num}}} # becomes 42\n"
```

Comment:
As I understand it, `raw` just gets the string `foo1`, not the value `${foo1}`.

Reply:
`raw` builds the string "foo1", and the outer braces become the expression `${foo1}` which indicates a replacement expression.  Without them, bar just becomes "foo1" instead of "42"

Reply:
Nice, thanks!